javascripts/scroll-up.js

```diff
@@ -0,0 +1,20 @@
+export default function () {
+  // function to scroll up to page top
+  const PageTopBtn = document.getElementById('js-scroll-top')
+  PageTopBtn.addEventListener('click', (e) => {
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth'
+    })
+  })
+
+  // show scroll button only when display is scroll down
+  window.addEventListener('scroll', function () {
+    const y = document.documentElement.scrollTop // get the height from page top
+    if (y < 100) {
+      PageTopBtn.classList.remove('show')
+    } else if (y >= 100) {
+      PageTopBtn.classList.add('show')
+    }
+  })
```

Comment:
I love this! Great addition 👏

Reply:
@JasonEtco I'm really glad to hear that✨